gradle/libs.versions.toml

```diff
@@ -1,30 +1,29 @@
 [versions]
-androidxCore = "1.12.0"
-androidxLifecycle = "2.7.0"
+androidxCore = "1.13.1"
+androidxLifecycle = "2.8.0"
 kotlinxCoroutinesTest = "1.7.3"
```

Comment:
i think we can also update this to `1.8.1`

https://mvnrepository.com/artifact/org.jetbrains.kotlinx/kotlinx-coroutines-test